packages/asset-service/src/service/descriptions.json

```diff
@@ -18,5 +18,6 @@
   "cosmos:cosmoshub-4/slip44:118": "Cosmos (ATOM) is a cryptocurrency that powers an ecosystem of blockchains designed to scale and interoperate with each other. The Cosmos vision aligns with the ShapeShift vision of a multi chain future, and enables a network of blockchains to be to communicate with each other in a decentralized manner. Cosmos is a proof-of-stake chain and allows holders to stake their ATOM to earn rewards using ShapeShift.",
-  "cosmos:osmosis-1/ibc:27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2": "Cosmos (ATOM) on Osmosis is a proof of stake cryptocurrency that powers an ecosystem of blockchains designed to interoperate and communicate with each other in a decentralized manner on the Osmosis blockchain, an alternative decentralized [IBC](https://tendermint.com/ibc/) blockchain."
+  "cosmos:osmosis-1/ibc:27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2": "Cosmos (ATOM) on Osmosis is a proof of stake cryptocurrency that powers an ecosystem of blockchains designed to interoperate and communicate with each other in a decentralized manner on the Osmosis blockchain, an alternative decentralized [IBC](https://tendermint.com/ibc/) blockchain.",
+  "eip155:1/erc20:0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d": "FOX is an ERC-20 token created by ShapeShift which serves as the governance token for the ShapeShift DAO. Token holders can vote on proposals relating to the operation and treasury of the DAO. The FOX token supports and provides utility for a borderless, cross-chain crypto trading platform and portfolio manager for user self-sovereignty. The ShapeShift ecosystem has been expanding rapidly around the new, decentralized, open source web app that allows users to not only trade and swap assets cross-chain, but also earn yield on the [DeFi tab](https://app.shapeshift.com/defi/earn) through staking assets across platforms such as Curve.Finance, Yearn.Finance, and the Cosmos ecosystem. Holders of the FOX token are also able to stake their FOX for ShapeShift's new FOXy token, where they will benefit from active yield management conducted by the ShapeShift DAO without any effort on their part. FOXy holders will receive additional FOXy generated from avenues such as the FOX Tokemak reactor, as well as a portion of revenue generated by the DAO through affiliate partnerships. The FOXy token is an only-up rebasing token always redeemable 1:1 with FOX, and is the best way to earn yield on FOX holdings. Users can deposit FOX for FOXy by clicking [here](https://app.shapeshift.com/#/defi/token_staking/ShapeShift/overview?chain=ethereum&contractAddress=0xee77aa3Fd23BbeBaf94386dD44b548e9a785ea4b&tokenId=0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d&rewardId=0xDc49108ce5C57bc3408c3A5E95F3d864eC386Ed3/)."

```

Comment:
We should use relative links for our internal links, see existing links for examples

Reply:
Should we use just [DeFi tab](https://app.shapeshift.com/defi/earn) instead of [here](https://app.shapeshift.com/#/defi/token_staking/ShapeShift/overview?chain=ethereum&contractAddress=0xee77aa3Fd23BbeBaf94386dD44b548e9a785ea4b&tokenId=0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d&rewardId=0xDc49108ce5C57bc3408c3A5E95F3d864eC386Ed3/)?

Reply:
Yes, all internal links should be relative - including the two in this line diff :)

Reply:
sounds clean mate, pushed, lmk

Reply:
@molecula451 I think there was a misunderstanding, please look at the review comment

Reply:
- `https://app.shapeshift.com/defi/earn` should be `/defi/earn`
- `https://app.shapeshift.com/#/defi/token_staking/ShapeShift/overview?chain=ethereum&contractAddress=0xee77aa3Fd23BbeBaf94386dD44b548e9a785ea4b&tokenId=0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d&rewardId=0xDc49108ce5C57bc3408c3A5E95F3d864eC386Ed3/` should be `/#/defi/token_staking/ShapeShift/overview?chain=ethereum&contractAddress=0xee77aa3Fd23BbeBaf94386dD44b548e9a785ea4b&tokenId=0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d&rewardId=0xDc49108ce5C57bc3408c3A5E95F3d864eC386Ed3` (please double check that it actually works like this)

Reply:
got it mate, tho i'm seeing links with the folder /assets/ using style you pointing, but actual link to websites are having the prefix <code> https</code>